### PR TITLE
Fixes #1159. Refactor DurabilityIT

### LINKED ISSUE #1159: Fix DurabilityIT
This IT is a performance IT which means its measuring performance but can fail quite frequently, depending on the situation.  The test testWriteSpeed is particularly flaky and should be rewritten. There are a few things that can be done to fix this IT.

1. Split out the Durability Unit Tests into a separate Test or IT
2. Drop the testWriteSpeed test and drop the performance aspect.

### PR DESCRIPTION
**Pull request by Manno15**
The performance aspect of this IT is being moved to accumulo-testing.

### CODE REVIEW

**Comment by milleruntime** [comment]
Sorry I didn't know the new test was only a performance Test.

**Comment by Manno15** [comment]
Sorry for the confusion guys. I should have asked for clarification on the issue ticket. I will make the changes when I get in tomorrow.

**Comment by Manno15** [comment]
I removed testWriteSpeed and I also change the writeSome function to void since no other function used the return value.

**Comment by keith-turner** [comment]
I am not sure @Manno15 .  I restarted the build, lets see what happens.

**Comment by Manno15** [comment]
@keith-turner Thanks for restarting it. It looks like it built this time. Not sure what happened the first time.

File: test/src/main/java/org/apache/accumulo/test/functional/DurabilityIT.java [hunk 1]
```diff
@@ -218,7 +192,7 @@ private void restartTServer() throws Exception {
     cluster.start();
   }
 
-  private long writeSome(AccumuloClient c, String table, long count) throws Exception {
+  private void writeSome(AccumuloClient c, String table, long count) throws Exception {
     int iterations = 5;
```
Comment (keith-turner) [inline test/src/main/java/org/apache/accumulo/test/functional/DurabilityIT.java]: @Manno15 could `iterations`, `attempts`, and the for loop be removed?  Seems like this loop only existed to write multiple times and get the median time.

Comment (Manno15) [reply test/src/main/java/org/apache/accumulo/test/functional/DurabilityIT.java]: Yeah, that appears to be the case. I'll remove it and then test it. Thanks

**Pull request merged**